src/Api/Models/Response/Organizations/OrganizationResponseModel.cs

```diff
@@ -81,6 +86,11 @@ public OrganizationResponseModel(Organization organization, string obj = "organi
     public bool UseCustomPermissions { get; set; }
     public bool SelfHost { get; set; }
     public bool HasPublicAndPrivateKeys { get; set; }
+    public bool UsePasswordManager { get; set; } = true;
+    public int? SmSeats { get; set; }
+    public int? SmServiceAccounts { get; set; }
+    public int? MaxAutoscaleSmSeats { get; set; } = null;
+    public int? MaxAutoscaleSmServiceAccounts { get; set; } = null;
```

Comment:
Specifying `null` as the default value is redundant here given the datatype is already an `int?`

Reply:
The default value is `0`, not `null`: https://learn.microsoft.com/en-us/dotnet/csharp/language-reference/builtin-types/default-values

I agree with your comment above though that these default assignments are probably not necessary.

Reply:
Oops sorry, the default value is indeed `null` for a nullable type. d'oh.

@cyprain-okeke This change is not showing, please double check that you've removed these assignments and pushed to remote.